docs/CONTRIBUTING.md

```diff
@@ -29,6 +29,7 @@ Here is what is needed:
 
 - `node` v14.x
 - `git` v2.x or greater
+- [`git-lfs`](https://git-lfs.github.com)
```

Comment:
I encountered a somewhat mysterious error when building the project on a clean slate. Turns out we need this too!